AdaSupport/embed
# EX-1161: Notification Badge for Embed

### What
* Adds new Notification Badge to Button
* `route` added to constructUrl
* iFrame is loaded initially that connects to `/chat/connect/` to retrieve chatterID
* API requests are made to retrieve and reset number of unread messages
* If chatter is in pending/active live state and Ada is not set to open in new window, chat is automatically started (although will be hidden)
### Testing Reset
* Make sure chatter is created, then set `chatter.unread_count` to above 0 in db
* Reload page, notice badge appear
* Open chat, check db as unread is set to 0
* Reset unread in db above 0, and close chat, unread should be 0 again
### Testing Incoming Messages
* Open chat, talk to bot or agent
* Close chat, if bot close before it responds, if agent respond after closing
* Notice as messages come in the badge will appear
### Testing Live State Active Chat
* Open chat and enter live agent mode (pending/active)
* Refresh, and open Chat (should have already loaded)
### Deployment
* Nothing special, all required API/CHAT components already shipped
---
